scripts/utils/InsertNuGetIntoSdkAndCli.ps1

```diff
@@ -34,7 +34,9 @@ param
     [Parameter(Mandatory=$True)]
     [string]$BranchName,
     [Parameter(Mandatory=$True)]
-    [string]$FilePath,
+    [string]$FilePath2,
```

Comment:
Why name is filepath2 and filepath3? 

I'd choose better names for that. 

Also can you link to the accompanying change in the release definition